indico/modules/rb/forms/settings.py

```diff
@@ -32,6 +32,7 @@
 class SettingsForm(IndicoForm):
     admin_principals = PrincipalListField(_('Administrators'), groups=True)
     authorized_principals = PrincipalListField(_('Authorized users/groups'), groups=True)
+    excluded_categories = MultipleItemsField('Excluded categories', fields=[{'id': 'id', 'caption': 'Category ID'}])
```

Comment:
needs a description, "Disable quick-book on event creation in these categories" or something similar